src/helics/application_api/Federate.cpp

```diff
@@ -9,6 +9,7 @@ SPDX-License-Identifier: BSD-3-Clause
 #include "../core/CoreFactory.hpp"
 #include "../core/core-exceptions.hpp"
 #include "../core/helics_definitions.hpp"
+#include "../core/CommonCore.hpp"
```

Comment:
The federate code should not need to be aware of CommonCore to maintain the layers properly.   All federate interaction should be with the interface in the core api (Core.hpp), otherwise we will run into a lot of trouble in the future making changes.

Reply:
This was accidentally left in. It is not necessary, and will be removed.